web/src/components/profile/ProfileErrorAlert.test.tsx

```diff
@@ -37,4 +37,11 @@ describe("<ProfileErrorAlert/>", () => {
       within(profileAlert).getByText(Config.profileDefaults.fields.petCareHousing.default.header)
     ).toBeInTheDocument();
   });
+
+  it("displays formation address error", () => {
```

Comment:
[sand] Can we change the name of the test to better reflect the behavior here given this is really confirming we're able to grab a field label from the config, not specifically related to address.